src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -519,4 +524,18 @@ private void verifyEmailReceivedCorrectly(
         assertTrue(hasReceivedEmailCorrectly);
     }
 
+    private void verifyEmailWithoutSpecifiedLineReceivedCorrectly(
```

Comment:
**To the reviewer:** I could not find a way to retrieve the reminder email sent to the instructor any other way, so Added this method, which checks for an email, which **does not** contain the specified line that is passed in the argument. (Trying to retrieve the email with using `verifyEmailReceivedCorrectly` retrieves the student email copy sent to the instructor, not the reminder email for the instructor )

Then searched for the email that does not have the line `=== Email message as seen by the students ===` (As this is the actual email the instructor receives to submit responses)

If there is any better way to retrieve this email, I will be happy to modify my logic accordingly.

Reply:
Hi @sukanta-27, since there is a `course` created for this test, you can try using `logic.getInstructorForEmail(String courseId, String email)` and check whether it returns you a null value. If it doesn't, that means the recipient of the email is an instructor.

Reply:
@LuLechuan I have some confusion with your comment. And that is, there is no problem in getting the instructor's email. 

https://github.com/TEAMMATES/teammates/blob/968c58b351fce9c892a5c20ad0af5bc9713ec0a3/src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java#L62-L68

My problem is, there are two emails sent to the instructor, 
the first one is the copy of the student reminder email. [screenshot]
![screenshot 473](https://user-images.githubusercontent.com/10681755/38165810-13ff8cba-3509-11e8-83c0-82cefca8cb6a.png)

And the second one is the reminder email that is sent to the instructor to remind him to complete the feedback session.
![screenshot 474](https://user-images.githubusercontent.com/10681755/38165837-79dc6012-3509-11e8-90db-d90ea7c08d05.png)

The problem that I was facing is that, I could not retrieve the second email sent to the instructor. The first email is retrieved using the method 
`private void verifyEmailReceivedCorrectly(List<EmailWrapper> actualEmails, String recipient, String subject, String emailContentFilePath, String containsString)`
but the other `verifyEmailReceivedCorrectly` method also retrieves the first email. So that is why I added this method, to retrieve the second email.

Reply:
I see, misinterpreted your question.

Reply:
This method should not be necessary. To hunt down the 2nd email, just look for that `/page/instructorFeedbackSubmissionEditPage?...` portion.

Reply:
Right!!! :man_facepalming: What was I thinking! :sweat_smile: